platform_services/test/test.js

```diff
@@ -209,4 +342,45 @@ describe('platform_services', function() {
     stub.restore();
     assert.isTrue(logCheck && cbCheck);
   });
```

Comment:
The changes look good for GET. Can you update this PR to submit to v1.1.1 (v1.1 is already merged into master and released)?